sample_apps/run_example.sh

```diff
@@ -1043,13 +1043,14 @@ function get_measurement_of_app_by_name() {
     if [ "${SampleAppName}" = "application_service" ]; then
         measurement_file="app_service.measurement"
     elif [ "${SampleAppName}" = "simple_app_python" ]; then
-        policy_key_arg="--policy_key=../policy_key.py"
+        policy_key_arg="--other_files=../policy_key.py,${CERT_PROTO}/certifier_framework.py,${CERT_PROTO}/libcertifier_framework.so"
```

Comment:
New arg `--other_files` to the measurement utility is applicable only for Python simple_app, here.